templates/quickstart-confluence-master.template.yaml

```diff
@@ -1664,7 +1670,7 @@ Resources:
         DBMasterUserPassword: !Ref DBMasterUserPassword
         DBMultiAZ: !Ref DBMultiAZ
         DBSecurityGroup: !Ref SecurityGroup
-        DBStorageEncrypted: !Ref DBStorageEncrypted
+        DBStorageEncrypted: !Ref EncryptionAtRest
```

Comment:
This will generate a unique key for the database. I don't think it really matters as the customers shouldn't be using the keys directly. But I thought I'd raise it in case it does matter?